src/index.js

```diff
@@ -19,7 +19,7 @@ export {
   FormattedNavLink,
   FormattedRedirect,
   I18nRoutingProvider,
-  withI18nRouting,
```

Comment:
Is okey to have `LocaleContext` children as a func but I would not remove `with18nRouting`. with18nRouting provides more than just the locale, exposing for example `setTranslatedRoutes`.

Having both, the HOC and children as func is the best to allow developers to use whatever that fits best to them.

Reply:
I'd want to see how many times `withI18nRouting` HOC is needed. One of the purpose of this library is to keep it simple, so expose things beyond our means it can become in a pain in the ass for the future; even so, I'll re-add HOC :)